sdk/videoanalyzer/pom.xml

```diff
@@ -7,8 +7,7 @@
   <groupId>com.azure</groupId>
   <artifactId>azure-videoanalyzer-service</artifactId>
   <packaging>pom</packaging>
-  <version>1.0.0</version><!-- Need not change for every release-->
-
+  <version>1.0.0-beta.1</version>  <!-- Need not change for every release-->
```

Comment:
This can be 1.0.0 as this isn't going to be released to Maven central and won't change.